paper/paper.md

```diff
@@ -0,0 +1,181 @@
+standardize the process of representing brain imaging data. BIDS is
+deliberately designed with adoption in mind; it adheres to a user-focused
+philosophy that prioritizes common use cases and discourages complexity. By
+successfully encouraging a large and ever-growing subset of the community to
+adopt a common standard for naming and organizing files, BIDS has made it much
+easier for researchers to share, re-use, and process their data
+[@Gorgolewski2017-cz].
+
+The ability to efficiently develop high-quality spec-compliant applications
+itself depends to a large extent on the availability of good tooling.
+Because many operations recur widely across diverse contexts—for example,
+almost every tool designed to work with BIDS datasets involves regular
+file-filtering operations—there is a strong incentive to develop utility
+libraries that provide common functionality via a standardized, simple API.
+
+``PyBIDS`` [@zenodo] is a Python package that makes it easier to work with BIDS
+datasets. In principle, its scope includes virtually any functionality that is
+likely to be of general use when working with BIDS datasets (i.e., that is not
+specific to one narrow context). At present, its core and most widely used
+module supports simple and flexible querying and manipulation of BIDS datasets.
+PyBIDS makes it easy for researchers and developers working in Python to search
+for BIDS files by keywords and/or metadata; to construct BIDS-valid path names
+for new files; and to validate projects against the BIDS specification, among
+other applications.
+
+In addition to this core functionality, PyBIDS also contains an ever-growing
+set of modules that support additional capabilities. At present, PyBIDS
+includes tools for (1) reading and manipulating data contained in various
+BIDS-defined files (e.g., physiological recordings, event files, or
+participant-level variables); (2) constructing design matrices and contrasts
+that support the new ``BIDS-StatsModel`` specification (for machine-readable
+representation of fMRI statistical models); and (3) automated generation of
+partial methods sections.
+
+PyBIDS can be easily installed on all platforms via pip (``pip install
+pybids``), though currently it is not officially supported on Windows. The
+package has few dependencies outside of standard Python numerical and image
+analysis libraries (i.e., numpy, scipy, pandas, and NiBabel). The core API
+is deliberately kept minimalistic: nearly all interactions with PyBIDS
+functionality occur through a core ``BIDSLayout`` object initialized by passing
+in a path to a BIDS project. For most applications, no custom configuration
+should be required.
+
+Although technically still in alpha release, PyBIDS is already being used both
+as a dependency in dozens of other open-source brain imaging packages--e.g.,
+fMRIPrep, MRIQC, and fitlins--and directly in many researchers' custom Python
```

Comment:
Would hurt to add references to all those projects. datalad-neuroimaging also uses it

Reply:
How do I cite datalad? Do my eyes deceive me, or is there *gasp* no duecredit support?